build.sbt

```diff
@@ -48,7 +48,6 @@ lazy val microservice = Project(appName, file("."))
   .configs(IntegrationTest)
   .settings(inConfig(IntegrationTest)(Defaults.itSettings): _*)
   .settings(resolvers += Resolver.jcenterRepo)
-  .settings(integrationTestSettings())
```

Comment:
We removed that in stubs, should we remove it in frontend?

I thought that in frontends we follow the migration guide and replace it with:
`.configs(ItTest)` & `.settings(inConfig(ItTest)(Defaults.testSettings): _*)` and so on

Reply:
I can try out that replacement - didn't see that part of the guide